### ISSUE TITLE
Narrow form of # months (#mo) is untranslatable to Italian

### ISSUE THREAD

**Issue by mlocati** [bluesky-social/social-app#6817]
### Steps to Reproduce

When viewing posts that are months old, Bluesky presents a narrow form of the "number of months".

Here's an example that displays "1 month" in this narrow form:

![immagine](https://github.com/user-attachments/assets/fb931724-2cad-4f6e-a4d5-f440bbd4520a)

In Italian we don't have a narrow form for months, probably because it's already rather short (singular: *mese*, plural: *mesi*).

The problem is that Bluesky doesn't support specifying plural forms - see
https://github.com/bluesky-social/social-app/blob/1.95.0/src/lib/hooks/useTimeAgo.ts#L170

- `{0}mese` doesn't work for > 1 months (it's like reading "2month" in English)
- `{0}mesi` doesn't work for 1 month (it's like reading "1months" in English)

At the moment we have `{0}mesi`, but it's really ugly to see this:

![immagine](https://github.com/user-attachments/assets/b49d1f93-a98b-495d-a1ee-489ba0045260)

### What platform(s) does this occur on?

Android, Web (Desktop)

### What version of the app are you using?

Build version: 1.95.0; Bundle info: c9d86de (prod); Bundle date: 24112719; Platform: web; Platform version: undefined

**Comment by c960657** [comment]
FWIW, [CLDR](https://cldr.unicode.org/) just uses the full form, `{0} mese`/`{0} mesi`.

https://github.com/unicode-org/cldr/blob/57e30cab4/common/main/it.xml#L2466-L2469

**Comment by mlocati** [comment]
Yep, supporting plural forms would solve the issue. But also avoiding the "X months ago" approach and instead displaying a date would work.
